src/app/shared/pipes/fy-currency.pipe.ts

```
@@ -20,6 +20,10 @@ export class FyCurrencyPipe implements PipeTransform {
     digitsInfo?: string,
     locale?: string
   ): string | null {
+    // When amount passed is 0 avoid adding decimal point
+    if (value === 0) {
+      digitsInfo = '1.0-0';
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)